package.json

```diff
@@ -16,9 +16,9 @@
     },
     "license": "MIT",
     "scripts": {
-        "test": "make test"
+        "test": "./node_modules/mocha-phantomjs/bin/mocha-phantomjs test/index.html"
```

Comment:
No need for the `./node_modules/mocha-phantomjs/bin/` part. It's one of the good thing with doing node script, it automatically search for binaries in the node_modules folder using node resolution pattern.
So basically : `mocha-phantomjs test/index.html`